src/js/View.js

```diff
@@ -581,7 +581,7 @@ View = (function() {
             }
 
             var xymouse = view.imageCanvas.relMouseCoords(e);
-            if (!view.dragging || hasTouchEvents) {
```

Comment:
Having hasTouchEvents here confuses me. I don't see the reason touch devices always should enter this if statement, while other devices only enter if they are not currently draggin

Reply:
By looking at it, I find it strange too, I think for touch move, one would need to update the box location but that's all. No need to do the callback things afterwards.